src/theme/ItaliaTheme/Subsites/ItaliaTheme/_common.scss

```diff
@@ -16,11 +27,269 @@
+  }
+
+  .bg-secondary {
+    background-color: $subsite-secondary !important;
+    color: $subsite-secondary-text;
+  }
+
+  //btn mini
+  .select-all-cts.btn.btn-link.btn-mini,
+  .subsite-header .text a.select-all-cts.btn-link.btn-mini {
+    color: $subsite-primary !important;
+  }
+
+  //icons - general
+  .icon-primary {
+    @if $subsite-light-theme {
+      fill: $subsite-secondary !important;
+    } @else {
+      fill: $subsite-primary !important;
+    }
+  }
+
+  // icon external link
+  .external-link {
+    @if $subsite-light-theme {
+      fill: $subsite-link-color !important;
+    } @else {
+      fill: $subsite-primary !important;
+    }
+  }
+
+  .flag-icon {
+    background: $subsite-secondary;
+  }
+
+  // color button widget - sidebar
+
+  .color-list-widget .button.primary,
+  .color-list-widget .button.primary.active {
+    background-color: $subsite-primary;
+  }
+
+  .color-list-widget .button.secondary,
+  .color-list-widget .button.secondary.active {
+    background-color: $subsite-secondary;
+  }
+
+  .color-list-widget .button.active {
+    border: 2px solid $subsite-primary;
+  }
+
+  //pagination - no background
+  @if $subsite-light-theme {
+    .page-link {
+      color: $subsite-link-color !important;
+
+      svg.icon-primary {
+        fill: $subsite-link-color !important;
+      }
+    }
+
+    .page-link[aria-current] {
+      border-color: $subsite-link-color !important;
+      color: $subsite-link-color !important;
+    }
+  } @else {
+    .page-link {
+      color: $subsite-primary !important;
+    }
+
+    .page-link[aria-current] {
+      border-color: $subsite-primary !important;
+      color: $subsite-primary !important;
+    }
+  }
+
+  // darken pagination when used on listing block with .background-image
+  .bg-light {
+    @if $subsite-light-theme {
+      .page-link {
+        color: darken($subsite-link-color, 5%) !important;
+
+        svg.icon-primary {
+          fill: darken($subsite-link-color, 5%) !important;
+        }
+      }
+
+      .page-link[aria-current] {
+        border-color: darken($subsite-link-color, 5%) !important;
+        color: darken($subsite-link-color, 5%) !important;
+      }
+    } @else {
+      .page-link {
+        color: darken($subsite-primary, 5%) !important;
+      }
+
+      .page-link[aria-current] {
+        border-color: darken($subsite-primary, 5%) !important;
+        color: darken($subsite-primary, 5%) !important;
+      }
+    }
+  }
+
+  // text - callout style
+  .callout-bg {
+    @if $subsite-light-theme {
+      background-color: $subsite-primary-a0;
+    } @else {
+      background-color: $subsite-primary-c1;
+    }
+  }
+
+  // text - quote style for view and edit
+  .block,
+  & {
+    blockquote {
+      border-left: 4px solid $subsite-secondary;
+    }
+  }
+
+  // titles and link colors - general
+  .card-title a,
+  .accordion-header button,
+  .category-top span.text span.text.fw-bold {
+    color: $subsite-link-color !important;
```

Comment:
È proprio obbligatorio avere questo `!important`? Occhio anche che important in questi casi di solito vince anche su `:hover` e `:focus` quindi di solito tocca cambiare anche quelli quando si cambia il colore del link

Reply:
Fixed per correggere la parte di highlitedContent